code/gateway/dataaccess.q

```diff
@@ -134,23 +147,6 @@ attributesrouting:{[options;procdict]
     :types;
     };
 
-// mixture of all the post processing functions in gw
-returntab:{[input;tab;reqno]
```

Comment:
So this function isn't used anymore anywhere?

Reply:
No, it seems to only be defined here and not used anywhere